src/SFML/Window/CMakeLists.txt

```diff
@@ -82,29 +82,47 @@ if(SFML_OS_WINDOWS)
     # make sure that we use the Unicode version of the Win API functions
     add_definitions(-DUNICODE -D_UNICODE)
 elseif(SFML_OS_LINUX OR SFML_OS_FREEBSD OR SFML_OS_OPENBSD OR SFML_OS_NETBSD)
-    set(PLATFORM_SRC
-        ${SRCROOT}/Unix/CursorImpl.hpp
-        ${SRCROOT}/Unix/CursorImpl.cpp
-        ${SRCROOT}/Unix/ClipboardImpl.hpp
-        ${SRCROOT}/Unix/ClipboardImpl.cpp
-        ${SRCROOT}/Unix/Display.cpp
-        ${SRCROOT}/Unix/Display.hpp
-        ${SRCROOT}/Unix/InputImpl.cpp
-        ${SRCROOT}/Unix/InputImpl.hpp
-        ${SRCROOT}/Unix/SensorImpl.cpp
-        ${SRCROOT}/Unix/SensorImpl.hpp
-        ${SRCROOT}/Unix/VideoModeImpl.cpp
-        ${SRCROOT}/Unix/VulkanImplX11.cpp
-        ${SRCROOT}/Unix/VulkanImplX11.hpp
-        ${SRCROOT}/Unix/WindowImplX11.cpp
-        ${SRCROOT}/Unix/WindowImplX11.hpp
-    )
     if(NOT SFML_OS_ANDROID)
-        set(PLATFORM_SRC
-            ${PLATFORM_SRC}
-            ${SRCROOT}/Unix/GlxContext.cpp
-            ${SRCROOT}/Unix/GlxContext.hpp
-        )
+        if(SFML_USE_DRM)
```

Comment:
@substring Could you take a look at this please?

Reply:
It appears this was a zombie conditional that was always evaluated as true for Linux and never reached for Android. @eXpl0it3r So what do we do? Shall we remove that  `if(NOT SFML_OS_ANDROID)` and move GlXContext up with the rest of Linux files?

Reply:
Android uses the EGLContext, so I guess this can be removed and doesn't need to be included for Android 

https://github.com/SFML/SFML/blob/c5f3aeca729b42e6381f3bcd59be536d46e74b4d/src/SFML/Window/GlContext.cpp#L85-L89